param/__init__.py

```diff
@@ -1544,8 +1578,26 @@ def serialize(cls, value):
 
     @classmethod
     def deserialize(cls, value):
-        from pandas import DataFrame as pdDFrame
-        return pdDFrame(value)
+        import pandas
+        try:
+            # FIXME(sdrobert): pandas.read_hdf5 requires pytables which can be
+            # installed no prob with conda but requires hdf5 headers for pip.
```

Comment:
I'd be inclined to add HDF5 to the list anyway, and just put the installation info into the docs, because if the user _has_ installed it it will work, and that's all we can say about _any_ of these options...

Reply:
You're okay with not having an associated test?

Reply:
A test skipped by default sounds perfect.

Reply:
Done